circuit/src/commonMain/kotlin/com/slack/circuit/CircuitInject.kt

```diff
@@ -18,6 +18,7 @@ package com.slack.circuit
 import kotlin.reflect.KClass
 
 // TODO: doc this
-@Suppress("unused") public annotation class CircuitInject<out Screen>
-
-public annotation class CircuitScope(val scope: KClass<*>)
+public annotation class CircuitInject(
```

Comment:
Can we add the kdoc for this now since this PR puts this in a pretty complete state.

Reply:
e75b6c8